keyserver/storage/tables/stale_device_lists_test.go

```diff
@@ -0,0 +1,94 @@
+package tables_test
```

Comment:
Do we want to use `package table` or `package tables_test` in future table tests?

Reply:
Not `table`. If you meant `tables` vs `tables_test` then it depends. The former exposes private functions, the latter doesn't. Given they are unit tests, am happy to expose private functions.